wandb/__init__.template.pyi

```diff
@@ -100,7 +101,7 @@ __version__: str = "0.18.4.dev1"
 run: Run | None
 config: wandb_config.Config
 summary: wandb_summary.Summary
-Api: PublicApi
+Api: Type[PublicApi]
```

Comment:
we use `from __future__ import annotations`, so i believe you can just use `type[PublicApi]`. also if you don't mind fixing the config/summary that would be awesome :)

Reply:
Yes, no problem